breakpad-symbols/src/sym_file/parser.rs

```diff
@@ -510,21 +521,18 @@ impl SymbolParser {
             Line::Function(mut cur, lines) => {
                 cur.lines = lines
                     .into_iter()
-                    .map(|l| {
+                    .filter(|l| {
                         // Line data from PDB files often has a zero-size line entry, so just
                         // filter those out.
-                        if l.size > 0 {
-                            if let Some(end) = l.address.checked_add(l.size as u64 - 1) {
-                                return (Some(Range::new(l.address, end)), l);
-                            }
-                        }
-
-                        (None, l)
+                        l.size > 0
```

Comment:
I was like "why the fuck did I write this like this" and tried to follow the blame and I think this is just ancient code that survived a million refactors.